plugins/genesys-gl32xx/genesys-gl32xx.quirk

```diff
@@ -5,3 +5,11 @@ FirmwareSize = 0x10000
 [BLOCK\VEN_05E3&DEV_0764]
 Plugin = genesys_gl32xx
 FirmwareSize = 0x1C000
+
+[BLOCK\VEN_32AC&DEV_0009]
+Plugin = genesys_gl32xx
+FirmwareSize = 0x1C000
+Name = Framework SD Expansion Card
+Vendor = Framework
+# Compatible with Genesys PID 0x0764
+CompatibleModel = 0x0764
```

Comment:
This needs to be the slightly verbose `GenesysGl32xxCompatibleModel` and you need to do `fu_context_add_quirk_key(ctx, "GenesysGl32xxCompatibleModel");` in the plugin.c too -- otherwise the daemon will get mad at you.